index.js

```diff
@@ -1,17 +1,60 @@
 class SortedList {
-  constructor() {}
+  constructor() {
+    this.items = [];
+    this.length = 0;
 
-  add(item) {}
+  }
 
-  get(pos) {}
+  add(item) {
+    this.items.push(item);
```

Comment:
Sabes que devuelve el metodo push :D el length del array despues de haberlo metido! es decir que puedes ahorrarte la linea 11 y poner this.length = this.items.push(item)

:D